plugins/org.jkiss.dbeaver.ext.tidb/plugin.xml

```diff
@@ -0,0 +1,139 @@
+                            <items label="%tree.partition.node.name" path="partition" property="partitions" icon="#partition">
+                                <items label="%tree.subpartitions.node.name" itemLabel="%tree.subpartition.node.name" path="subpartition" property="subPartitions" navigable="false" inline="true"/>
+                            </items>
+                        </folder>
+                    </items>
+                </folder>
+                <folder type="org.jkiss.dbeaver.ext.mysql.model.MySQLView" label="%tree.views.node.name" icon="#folder_view" description="%tree.views.node.tip">
+                    <items label="%tree.view.node.name" path="view" property="views" icon="#view">
+                        <items label="%tree.columns.node.name" itemLabel="%tree.column.node.name" path="column" property="attributes" icon="#column"/>
+                    </items>
+                </folder>
+                <folder type="org.jkiss.dbeaver.model.struct.rdb.DBSTableIndex" label="%tree.indexes.node.name" icon="#indexes" description="%tree.indexes.node.tip" virtual="true">
+                    <items label="%tree.index.node.name" path="index" property="indexes" icon="#index" virtual="true">
+                        <items label="Index columns" itemLabel="%tree.column.node.name" path="column" property="attributeReferences" navigable="false" inline="true" virtual="true"/>
+                    </items>
+                </folder>
+                <folder type="org.jkiss.dbeaver.ext.mysql.model.MySQLSequence" label="%tree.sequences.node.name" icon="#sequences" description="%tree.sequences.node.tip" visibleIf="object.dataSource.supportsSequences()">
+                    <items label="%tree.sequences.node.name" path="sequence" property="sequences" icon="#sequence">
+                    </items>
+                </folder>
+            </treeInjection>
+
+            <drivers managable="true">
+                 <driver
+                    id="tidb"
+                    label="TiDB"
+                    icon="icons/tidb_icon.png"
+                    iconBig="icons/tidb_icon_big.png"
+                    class="com.mysql.cj.jdbc.Driver"
+                    sampleURL="jdbc:mysql://{host}[:{port}]/[{database}]"
+                    defaultPort="4000"
+                    defaultUser="root"
+                    webURL="https://pingcap.com/"
+                    propertiesURL="https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference-configuration-properties.html"
+                    description="TiDB Driver (MySQL Connector/J 8.0)"
+                    promoted="1"
+                    categories="sql">
+
+                    <file type="jar" path="maven:/mysql:mysql-connector-java:RELEASE" bundle="!drivers.mysql"/>
+                    <file type="license" path="drivers/mysql/LICENSE.txt" bundle="drivers.mysql"/>
```

Comment:
`<file type="license" path="https://www.gnu.org/licenses/old-licenses/lgpl-2.0.txt" bundle="!drivers.mysql"/>`

Please, add the path to the license here